src/services/logging/sentry/log.ts

```diff
@@ -38,6 +38,7 @@ export const info = (message: string, severity: typeof debugLevel | typeof infoL
 export const log404NotFound = () => {
   Sentry.withScope(scope => {
     scope.setLevel(errorLevel);
-    Sentry.captureEvent({ message: `404: ${document.location.href}` });
+    const message = `404: '${document.location.href}'` + (document.referrer ? ` Referrer: '${document.referrer}'` : '');
```

Comment:
Not sure if there is a better way to add an extra field to the sentry event and send together the url and the referer if there's any